XamlControlsGallery/ControlPages/CommandBarPage.xaml

```diff
@@ -52,7 +52,7 @@
                 <CommandBar.SecondaryCommands>
                     <AppBarButton x:Name="settingsButton" Icon="Setting" Label="Settings">
                         <AppBarButton.KeyboardAccelerators>
-                            <KeyboardAccelerator Modifiers="Control" Key="S" />
+                            <KeyboardAccelerator Modifiers="Control" Key="I" />
```

Comment:
This change should also be present in the code sample below (at line 82)

Reply:
ah yeah, thanks!